WordPress/src/main/res/values/styles.xml

```diff
@@ -687,4 +687,52 @@
         <item name="android:maxHeight">8dip</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <!--Plugin Styles-->
+    <style name="PluginCardView">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginBottom">@dimen/margin_extra_large</item>
+    </style>
+
+    <style name="PluginCardViewVerticalContainer">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:orientation">vertical</item>
+    </style>
+
+    <style name="PluginCardViewHorizontalContainer">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:padding">@dimen/margin_extra_large</item>
+    </style>
+
+    <style name="PluginCardViewDivider">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">@dimen/list_divider_height</item>
+        <item name="android:background">@color/grey_lighten_20</item>
+    </style>
+
+    <style name="PluginCardViewPrimaryText">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_centerVertical">true</item>
+        <item name="android:textColor">@color/grey_dark</item>
+        <item name="android:textSize">@dimen/text_sz_large</item>
+    </style>
+
+    <style name="PluginCardViewSecondaryElement">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_centerVertical">true</item>
+        <item name="android:layout_alignParentEnd">true</item>
+        <item name="android:layout_alignParentRight">true</item>
+    </style>
+
+    <style name="PluginCardViewSecondaryElement.ExternalLinkImage" parent="PluginCardViewSecondaryElement">
+        <item name="android:layout_width">16dp</item>
```

Comment:
Could we use `@dimen/margin_extra_large` for these two?

Reply:
I considered the same thing and there are 2 reasons I didn't opt to do that:
1. I think it feels really weird to assign a margin value to width of an element
2. For consistency. As far as I can tell, we never do that ^, but instead assign a custom value instead, even if it's the same value.
Bonus: I actually think this is a better practice. Not that it'll actually happen, but if we were to ever change those base values, I don't think we actually want to change these item properties as well.

But then again, I really don't like having a static number there. So with all of that said, I'll leave the call to you :)

Reply:
> if we were to ever change those base values, I don't think we actually want to change these item properties as well.

That's a good point. In that case let's extract it to a new `dimen` resource and use that here.

Reply:
Done in b952b1d9bbc303284b33fe22d6eff4ad6b8d3643